.github/workflows/ci-additional.yaml

```diff
@@ -3,9 +3,11 @@ on:
   push:
     branches:
       - "main"
+      - "backend-indexing"
```

Comment:
Are these supposed to be here?

Reply:
I'm guessing that's what "delete CI changes" in the checklist refers to